src/sass/layouts/_footer.scss

```diff
@@ -1,2 +1,30 @@
 @use '../utilities/variables';
 
+.footer__container {
+  height: 100px;
+  width: 100%;
+  background-color: #f7f7f7; // change to variable
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
```

Comment:
I understand the position absolute and bottom 0 to keep the footer at the bottom of the page but it is not necessary. Feedback is along the lines of the body styles comment.

Reply:
Why is it not necessary? If i remove this style we will have the footer on the top of the page. For example,  when you open your watchlist for the first time it is empty and the footer will be in the middle of the page.
<img width="1470" alt="Screenshot 2023-04-13 at 10 39 24 AM" src="https://user-images.githubusercontent.com/112436831/231812458-299c91cd-b286-43eb-968b-d906e6d0f6a2.png">

Reply:
Thats, fine for now. As The header and other code gets pushed things will fall into place. We do not need any code not necessary to the finished project merged together.